modules/tensor_mechanics/include/materials/SingleVariableReturnMappingSolution.h

```diff
@@ -112,6 +112,12 @@ class SingleVariableReturnMappingSolutionTempl
   virtual Real computeReferenceResidual(const GenericReal<is_ad> & effective_trial_stress,
                                         const GenericReal<is_ad> & scalar) = 0;
 
```

Comment:
Maybe good to document here the use case

Reply:
👍